client/rest-high-level/src/main/java/org/elasticsearch/client/indices/PutComponentTemplateRequest.java

```diff
@@ -78,6 +78,10 @@ public PutComponentTemplateRequest componentTemplate(ComponentTemplate component
         return this;
     }
 
+    public ComponentTemplate componentTemplate() {
```

Comment:
is this used anywhere?

Reply:
I think this is a leftover, I'll remove it